.github/workflows/continuous-delivery.yaml

```diff
@@ -39,12 +39,19 @@ jobs:
           commit-assets: |
             ./dist
           node-module: true
+          release-rules:
+            '[ { "release": "patch", "type": "build" }, { "release": "patch",
+            "type": "chore", "scope": "deps" }, { "release": "patch", "type":
+            "chore", "scope": "deps-dev", "message": "*typescript*" }, {
+            "release": "patch", "type": "ci" }, { "release": "patch", "type":
+            "docs" }, { "release": "patch", "type": "improvement" }, {
+            "release": "patch", "type": "refactor" }, ]'
       - if: steps.release.outputs.released == 'true'
         name: Authenticate
         uses: actions/checkout@v2
         with:
-          fetch-depth: 1
           persist-credentials: true
+          ref: master
```

Comment:
I have trouble understanding why is this change necessary.

Reply:
This bled here from the other PR, which I would like merged (or closed) first .

Reply:
Removed!